src/git_changelog/cli.py

```diff
@@ -259,6 +287,76 @@ def _unreleased(versions: list[Version], last_release: str) -> list[Version]:
     return versions
 
 
+def read_config(
+    config_file: str | list[str] | None = DEFAULT_CONFIG_FILES,
+) -> dict:
+    """Find config files and initialize settings with the one of highest priority.
+
+    Arguments:
+        config_file: A path or list of paths to configuration file(s); or ``None`` to
+            disable config file settings. Default: ``pyproject.toml`` and ``.git-changelogrc``
```

Comment:
It's listed as `.git-changelog.toml` before, not `.git-changelogrc`.

Reply:
Thanks for spotting this. Will change.